config.md

```diff
@@ -317,6 +320,10 @@ Annotations are key-value maps.
 }
 ```
 
+## Extensibility
+The `annotations` property MAY be used as an extensibility point to include additional information that is not defined as part of this specification.
+Excluding the `annotations` property, implementations that are reading/processing this configuration file MUST generate an error if they encounter an unkown property.
```

Comment:
I'd prefer we either ignore or fault unknown properties - no reason to open up interoperability issues with SHOULDs/MAYs/etc...   I picked the fault path because that's where more people seemed to be headed during the OCI F2F last week.  So this issue (and https://github.com/opencontainers/image-spec/pull/164 ) were designed to have the discussion - once we pick one we just need to make the same decision in both spots.